### PR TITLE
feat(RHIF-229):replace beta with preview

### PR DESCRIPTION
**Pull request by adonispuente**
# Description
Replaces /beta with /preview 

# Dependent work link

# Checklist:

- [ ] The commit message has the Jira ticket linked
- [x] PR has a short description
- [ ] Screenshots before and after the change are added
- [ ] Tests for the changes have been added
- [x] README.md is updated if necessary
- [ ] Needs additional dependent work

### CODE REVIEW

**Review by mkholjuraev — APPROVED** [review]
LGTM!

File: config/dev.webpack.config.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ const INVENTORY_FRONTEND_PORT = '8003';
 
 const webpackProxy = {
   deployment: process.env.BETA ? 'beta/apps' : 'apps',
-  appUrl: process.env.BETA ? ['/beta/insights/advisor'] : ['/insights/advisor'],
+  appUrl: process.env.BETA
```
[the remark this answers is not in the document]
Comment (adonispuente) [reply config/dev.webpack.config.js:20]: For the time being arent be just replacing the url instances? If so we need to update all the other url prs. Its fine if we do but I just want to be sure thats the consensus

[the remark this answers is not in the document]
Comment (adonispuente) [reply config/dev.webpack.config.js:20]: I've updated the PR in accordance to the other merged PRS

**Comment by adonispuente** [comment]
@mkholjuraev good catch! Thanks for taking another look.

File: package.json [hunk 2]
```diff
@@ -72,10 +72,10 @@
     "@redhat-cloud-services/frontend-components-advisor-components": "^1.0.6",
     "@redhat-cloud-services/frontend-components-charts": "3.2.1",
     "@redhat-cloud-services/frontend-components-notifications": "^3.2.5",
-    "@redhat-cloud-services/frontend-components-pdf-generator": "2.6.7",
+    "@redhat-cloud-services/frontend-components-pdf-generator": "^2.6.7",
```
Comment (johnsonm325) [inline package.json:75]: If you update this to `^2.6.19`, you won't get those errors anymore.

**Review by johnsonm325 — DISMISSED** [review]

File: package.json (hunk 2 above)
Comment (adonispuente) [reply package.json:75]: Im going to add this in there!

**Pull request merged**